en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the you have a keyboard and monitor attached to the device or have a serial connection, you can use console-conf to configure the network and add a user account. The user account is created based on your registered email address and ssh key. This user account can then be used to ssh to the device for various management purposes. (Technically, this is not a system user account because it is not created through assertion, but the end result is equivalent.)
```

Comment:
Sorry for the intrusion but creating a verb out of a network protocol can provoke a squeamish response in some readers. I know people talk like this ("ssh to the host") but you might want to reconsider putting this in writing (suggestions: "connect via SSH"; "use SSH to connect; "open an SSH session"). If not, then I actually prefer the original 'ssh' to the proposed 'SSH' as the capital letters most definitely refer to the protocol and not a self-made verb.

Whereas "ssh key" should unquestionably be "SSH key".